sevntu-checkstyle-sonar-plugin/checkstyle.properties

```diff
@@ -0,0 +1,4 @@
+checkstyle.suppressions.file=suppressions.xml
+checkstyle.header.file=https://raw.githubusercontent.com/checkstyle/checkstyle/master/config/java.header
```

Comment:
@romani Please confirm you are ok with specified header. Are we using the same header for everything we own?